src/test/java/org/dependencytrack/policy/cel/persistence/CelPolicyDaoTest.java

```diff
@@ -109,7 +109,6 @@ public void testLoadRequiredFieldsForProject() throws Exception {
                           "name": "projectName",
                           "version": "projectVersion",
                           "classifier": "APPLICATION",
-                          "isActive": true,
```

Comment:
If I'm not mistaken this should be `"isActive": false` now, since `inactiveSince` is populated above.